src/components/Sponsors/Sponsors.tsx

```diff
@@ -13,11 +37,26 @@ export default function Sponsors(): React.JSX.Element {
 
   useEffect(() => {
     fetchData(colorMode, 'landing');
-  }, [colorMode]);
-
-  if (!data) {
-    return <div>Loading...</div>;
-  }
+  }, [colorMode, counter]);
 
-  return <div dangerouslySetInnerHTML={{ __html: sanitize(data) }} />;
+  return (
+    <div>
+      {data ? (
+        <div dangerouslySetInnerHTML={{ __html: sanitize(data) }} />
```

Comment:
```suggestion
        <div className="min-h-[90px]" dangerouslySetInnerHTML={{ __html: sanitize(data) }} />
```
Goes together with the CSS change